ng-package.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "./node_modules/ng-packagr/ng-package.schema.json",
-  "whitelistedNonPeerDependencies": ["."],
+  "whitelistedNonPeerDependencies": [".", "@types/gapi"],
```

Comment:
I'm curious what this is for?

Reply:
Because you have a global dependency on gapi, the library builder complains and demands that this be specified explicitly here. I think it's technically wrong, and I think the real fix is that our use of gapi should not depend on a HTML import, but instead be done by the service class. Here's an example: https://github.com/rubenCodeforges/ng-gapi#readme

Reply:
Ah, good point. Could you file an issue for that?